.github/readmes/typescript/postgis-express/README.md

```diff
@@ -27,3 +27,7 @@ __INLINE(./_using-the-api.md)__
 ### Seeding data
 
 Add data of your choice to the "User" and "Location" tables with the above endpoints and then run the `GET` endpoint passing the required parameters.
+
+__INLINE(../../_switching-databases.md)__
```

Comment:
Since this is a Postgres-specific example I think for this one we can spare the "switching databases" section

```suggestion
```